ERROR - 2023-05-31 01:09:37 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:09:37 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:09:37 --> Undefined array key "Detalle"
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "Detalle"', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:10:16 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:10:16 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:10:16 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:10:22 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:10:22 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:10:22 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:27:41 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:27:41 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:27:41 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:29:05 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:29:05 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:29:05 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:29:08 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:29:08 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:29:08 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:30:20 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:30:20 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:30:20 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:30:46 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:30:46 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:30:46 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:32:40 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:32:40 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:32:40 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:32:54 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:32:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:32:54 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:33:02 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:33:02 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:33:02 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:33:32 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:33:32 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:33:32 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:33:35 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:33:35 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:33:35 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:33:40 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:33:40 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:33:40 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:36:39 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:36:39 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:36:39 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:38:28 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:38:28 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:38:28 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:38:39 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:38:39 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:38:39 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:41:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:41:11 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:41:11 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:42:08 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:42:08 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:42:08 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:42:12 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:42:12 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:42:12 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:43:33 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Proveedores.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Proveedores->index()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Proveedores))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\breadandfred\app\Controllers\Proveedores.php(12): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Proveedores->index()
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Proveedores))
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-05-31 01:43:33 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `registros`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Proveedores.php(12): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Proveedores->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Proveedores))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:44:16 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:44:16 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:44:16 --> Undefined array key "Detalle"
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "Detalle"', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:46:20 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'restful'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:46:20 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:46:20 --> Undefined array key "Detalle"
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "Detalle"', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-05-31 01:47:52 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'restful' in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'restful', Object(SensitiveParameterValue), 'restful', 3305, '', 0)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#4 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#10 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database '****' in C:\xampp\htdocs\breadandfred\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\breadandfred\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\breadandfred\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `re...', Array, false)
#3 C:\xampp\htdocs\breadandfred\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\breadandfred\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\breadandfred\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\breadandfred\app\Controllers\Registros.php(59): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(934): App\Controllers\Registros->create()
#8 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
#9 C:\xampp\htdocs\breadandfred\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\breadandfred\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-05-31 01:47:52 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database '****'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `registros` (`nombres`, `apellidos`, `email`, `cliente_id`, `llave_secreta`) VALUES (:nombres:, :apellidos:, :email:, :cliente_id:, :llave_secreta:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registros.php(59): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Registros->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registros))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-05-31 01:47:52 --> Trying to access array offset on value of type null
in APPPATH\Views\welcome_message.php on line 62.
 1 APPPATH\Views\welcome_message.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\breadandfred\\app\\Views\\welcome_message.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('welcome_message', [], true)
 5 APPPATH\Controllers\Home.php(9): view('welcome_message')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
